src/wscript_build

```diff
@@ -1,17 +1,95 @@
+# Detect a Windows build.
+is_win = 'WIN32' in ' '.join(bld.env.keys())
+
+# Detect MSVC
+is_msvc = bld.env.CXX_NAME == 'msvc'
+
 all_sources = bld.path.ant_glob('*.cpp scene-ideas/*.cc scene-terrain/*.cpp')
 common_sources = [f for f in all_sources if f.name.find('canvas-') == -1 and
                                             f.name.find('android') == -1 and
                                             f.name.find('native-state-') == -1 and
                                             f.name.find('gl-state-') == -1 and
                                             f.name.find('main.cpp') == -1]
-common_uselibs = ['libpng']
+local_uselibs = ['libpng-local', 'zlib-local', 'libjpeg-turbo-local']
+
+common_uselibs = []
```

Comment:
Perhaps rename to platform_uselibs to be consistent with the newly added platform_libs and platform_includes.

Reply:
Done.